app/invocation/invocation_exec_log_card.tsx

```diff
@@ -112,7 +114,11 @@ export default class InvocationExecLogCardComponent extends React.Component<Prop
     }
 
     try {
-      rpcService.downloadBytestreamFile("execution.log", profileFile.uri, this.props.model.getInvocationId());
+      rpcService.downloadBytestreamFile(
+        "execution_log.binpb.zstd",
```

Comment:
doesn't this depend on which of the execution_log flags the user chose to use (json, binary, compact?)

Reply:
Currently only the compact format will get published via BES
```
sluongng/exec-log-extension ~/work/bazelbuild/bazel> rg shouldPublish -A3 src/main/java/com/google/devtools/build/lib/exec/
src/main/java/com/google/devtools/build/lib/exec/CompactSpawnLogContext.java
190:  public boolean shouldPublish() {
191-    // The compact log is small enough to be uploaded to a remote store.
192-    return true;
193-  }

src/main/java/com/google/devtools/build/lib/exec/ExpandedSpawnLogContext.java
138:  public boolean shouldPublish() {
139-    // The expanded log tends to be too large to be uploaded to a remote store.
140-    return false;
141-  }

src/main/java/com/google/devtools/build/lib/exec/SpawnLogContext.java
78:  public abstract boolean shouldPublish();
79-
80-  /** Computes the environment variables. */
81-  protected ImmutableList<EnvironmentVariable> getEnvironmentVariables(Spawn spawn) {
```

The other formats are usually quite chunky in size. (could goes up to hundred of gigs each build in some setup)